cmd/prometheus/config.go

```diff
@@ -334,7 +334,7 @@ func parsePrometheusURL() error {
 	}
 
 	if ok := govalidator.IsURL(cfg.prometheusURL); !ok {
-		return fmt.Errorf("invalid Prometheus URL: %s", cfg.prometheusURL)
+		return fmt.Errorf("invalid Prometheus URL: %s, allowed characters: a-z 0-9 .", cfg.prometheusURL)
```

Comment:
This is misleading, more characters than this are permitted in URLs (and underscores are permitted in domain names per the spec).